Docs: add hidden_users configuration field (#30435)
grafana/grafana: docs/sources/administration/configuration.md

This setting should be expressed as a duration. Examples: 6h (hours), 2d (days), 1w (week).
Default is `24h` (24 hours). The minimum supported duration is `15m` (15 minutes).

### hidden_users

> A comma-separated list of usernames to hide them in the Grafana UI. These users are shown to Grafana admins and to themselves.

Reviewer: ```suggestion
A comma-separated list of usernames to hide them in the Grafana UI. They are also visible to Grafana administrators and to themselves.
```

Reviewer 2: The first sentence is missing a subject and a verb. WHAT is the comma-separated list doing? Are you adding a list of names to hide from the UI? If so, why? Do we explain that somewhere?

> Set a comma-separated list of usernames to hide the corresponding users in the Grafana UI. They are still visible to Grafana administrators and to themselves.

Reviewer: ```suggestion
This is a comma-separated list of usernames. Users specified here are hidden in the Grafana UI. They are still visible to Grafana administrators and to themselves.
```
